### PR TITLE
docs(ES-845513): Include UG for editor header and footer.

### PR DESCRIPTION
**Pull request by AshokkumarKaruppasamy**
Include UG for editor header and footer.

### CODE REVIEW

File: ej2-javascript/schedule/editor-template.md [hunk 1]
```diff
@@ -254,6 +254,71 @@ By default, the timezone collections in the editor window have been loaded with
 {% previewsample "page.domainurl/code-snippet/schedule/editor-window-cs7" %}
 {% endif %}
 
+
+## Customizing event header using template
```
Comment (VengateshManiraj) [inline ej2-javascript/schedule/editor-template.md:258]: Customizing editor window header and footer using template

**Review by VengateshManiraj — CHANGES_REQUESTED** [review]

File: ej2-javascript/schedule/editor-template.md [hunk 2]
```diff
@@ -254,6 +254,39 @@ By default, the timezone collections in the editor window have been loaded with
 {% previewsample "page.domainurl/code-snippet/schedule/editor-window-cs7" %}
 {% endif %}
 
+
+## Customizing editor window header and footer using template
```
Comment (VengateshManiraj) [inline ej2-javascript/schedule/editor-template.md:258]: How to customize header and footer using template

File: ej2-javascript/schedule/editor-template.md [hunk 3]
```diff
@@ -254,6 +254,39 @@ By default, the timezone collections in the editor window have been loaded with
...
+The editor custom header can be customized by using the `editorHeaderTemplate` option, and editor window footer can be customized footer by making use of the `editorFooterTemplate` option. In this scenario, a custom header and footer design is built with the required fields using the script template and its type should be of **text/x-template**.
```
Comment (VengateshManiraj) [inline ej2-javascript/schedule/editor-template.md:259]: The editor window's header and footer can be enhanced with custom designs using the `editorHeaderTemplate` and `editorFooterTemplate` options. To achieve this, create a script template that includes the necessary fields. Ensure that the template type is set to `text/x-template`.

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/index.ts [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+let endDateTime: HTMLInputElement;
+let allDay: HTMLInputElement;
+let subject: HTMLInputElement;
+let isEmptyDelete: boolean = false;
+const scheduleObj: Schedule = new Schedule({
+    width: '100%',
+    height: '550px',
+    editorHeaderTemplate: '#editor-header',
+    editorFooterTemplate: '#editor-footer',
+    showQuickInfo: false,
```
Comment (VengateshManiraj) [inline ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/index.ts:41]: why showQuickInfo set as false

File: ej2-javascript/schedule/editor-template.md [hunk 5]
```diff
@@ -254,6 +254,39 @@ By default, the timezone collections in the editor window have been loaded with
...
+
+In this demo, the editor custom header is customized based on the appointment subject field which can be achieved by the `editorHeaderTemplate`.Additionally, the `editorFooterTemplate` is employed to handle the functionality for saving, deleting, and canceling actions.
```
Comment (VengateshManiraj) [inline ej2-javascript/schedule/editor-template.md:261]: In this demo, we customize the editor's header based on the appointment's subject field using the `editorHeaderTemplate`. Additionally, we utilize the `editorFooterTemplate` to manage the functionality for saving, deleting, and canceling actions.

**Review by VengateshManiraj — CHANGES_REQUESTED** [review]

File: ej2-javascript/schedule/editor-template.md (hunk 2 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/schedule/editor-template.md]: changed the header ange updated

File: ej2-javascript/schedule/editor-template.md (hunk 3 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/schedule/editor-template.md]: changed the content

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/index.ts (hunk 4 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/index.ts]: Addressed the corrections and changes updated

File: ej2-javascript/schedule/editor-template.md (hunk 5 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/schedule/editor-template.md]: changed the content

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-schedule/styles/material.css" rel="stylesheet">
+    <script src="https://cdn.syncfusion.com/ej2/20.4.38/dist/ej2.min.js" type="text/javascript"></script>
+</head>
+<script id="editor-header" type="text/x-template">
+    ${if (!Subject)}<div>Create New Event</div>${else}<div>${Subject}</div>${/if}
+</script>
+<script id="editor-footer" type="text/x-template">
+    <div id="verify">
+        <input type="checkbox" id="check-box" value="unchecked">
+        <label id="text">Verify</label>
```
Comment (VengateshManiraj) [inline ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html:27]: Verified

File: ej2-javascript/schedule/editor-template.md [hunk 7]
```diff
@@ -291,6 +291,39 @@ As we are using our Syncfusion sub-components within our editor using template i
 {% previewsample "page.domainurl/code-snippet/schedule/editor-cs1" %}
 {% endif %}
 
+## How to customize header and footer using template
+
+The editor window's header and footer can be enhanced with custom designs using the `editorHeaderTemplate`  and `editorFooterTemplate` options. To achieve this, create a script template that includes the necessary fields. Ensure that the template type is set to **text/x-template**.
+
+In this demo, we customize the editor's header based on the appointment's subject field using the `editorHeaderTemplate`. Additionally, we utilize the `editorFooterTemplate` to manage the functionality for saving, and canceling and verify actions. If the check box is checked, the save button is enabled, if it is unchecked, the save button is disabled.
```
Comment (VengateshManiraj) [inline ej2-javascript/schedule/editor-template.md:298]: In this demo, we tailor the editor's header according to the appointment's subject field using the `editorHeaderTemplate`. Furthermore, we make use of the `editorFooterTemplate` to handle the functionality of validating specific fields before proceeding with the save action or canceling it if validation requirements are not met.

**Review by VengateshManiraj — CHANGES_REQUESTED** [review]

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html (hunk 6 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html]: changed

File: ej2-javascript/schedule/editor-template.md (hunk 7 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/schedule/editor-template.md]: I've modified the content and made updates.

**Review by VengateshManiraj — APPROVED** [review]

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    <link href="index.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-base/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-buttons/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-calendars/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-dropdowns/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-inputs/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-navigations/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-popups/styles/material.css" rel="stylesheet">
+    <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-schedule/styles/material.css" rel="stylesheet">
+    <script src="https://cdn.syncfusion.com/ej2/20.4.38/dist/ej2.min.js" type="text/javascript"></script>
```
Comment (maheshtps) [inline ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html:19]: use the latest versions to avail this feature

**Review by maheshtps — CHANGES_REQUESTED** [review]

File: ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html (hunk 8 above)
Comment (AshokkumarKaruppasamy) [reply ej2-javascript/code-snippet/schedule/editor-header-footer-cs1/js/index.html]: i have addressed the review corrections and  changes updated

**Review by VengateshManiraj — APPROVED** [review]

**Review by maheshtps — APPROVED** [review]

**Pull request merged**